lib/facter/self_service.rb

```diff
@@ -109,4 +109,10 @@
     # Did catalog apply successfully on last puppet run
     { S0013: File.open(summary_path).read.include?('catalog_application') }
   end
+
+  chunk(:S0021) do
+    # Is there at least 9% memory available
+    next unless PuppetSelfService.replica? || PuppetSelfService.postgres? || PuppetSelfService.primary? || PuppetSelfService.compiler? || PuppetSelfService.legacy_compiler?
```

Comment:
This can probably be removed since we want to run it on on everything this fact is confined to.

Reply:
oh yeah....